[FIX] add paragraph on MEG specific "markers" suffix in MEG spec (#653)
bids-standard/bids-specification: src/04-modality-specific-files/02-magnetoencephalography.md

MAY be included alongside the MEG data; examples are provided below.

This template is for MEG data of any kind, including but not limited to
task-based, resting-state, and noise recordings.
If multiple *Tasks* were performed within a single *Run*,
the task description can be set to `task-multitask`.
The `*_meg.json` file SHOULD contain details on the *Tasks*.

Some manufacturers' data storage conventions use folders which contain data
files of various nature: for example, CTF's `.ds` format, or BTi/4D's data folder.
Yet other manufacturers split their files once they exceed a certain size
limit.
For example Neuromag/Elekta/Megin, which can produce several files
for a single recording.
Both `some_file.fif` and `some_file-1.fif` would belong to a single recording.
In BIDS, the [`split`](../99-appendices/09-entities.md#split) entity is RECOMMENDED to deal with split files.

> Another manufacturer specific detail pertains to the KIT/Yokogawa/Ricoh sytem,

Reviewer: ```suggestion
Another manufacturer-specific detail pertains to the KIT/Yokogawa/Ricoh sytem,
```

> which saves the MEG sensor coil positions in two separate files (`.sqd`, `,mrk`).

Reviewer 2: ```suggestion
which saves the MEG sensor coil positions in two separate files (`.sqd`, `.mrk`).
```

> which saves the MEG sensor coil positions in two separate files (`.sqd`, `,mrk`).

Reviewer 3: this should be clarified. They're not saved in two separate files; there are two different file extensions they can be saved to. The number of MEG sensor coil position files depend on the number of recordings the experimenter recorded

Author: Thanks @teonbrooks - as apparent from the vague text, I actually don't know all that much about KIT/Yokogawa/Ricoh.

So I'd be grateful for an expert to chime in and edit the text.

If nobody has time I'll have to search a user's manual and do it myself + hope for the best :)

> which saves the MEG sensor coil positions in two separate files (`.sqd`, `.mrk`).

Reviewer 3: ```suggestion
which saves the MEG sensor coil positions in a separate file with two possible filename extensions  (`.sqd`, `.mrk`).
```

Author: do you know which factors determine the extension? For example are `.sqd` raw files always associated with `*_markers.sqd` and `.con` raw files always with `*_markers.mrk`?

And do you know what `.con` stands for? `.sqd` probably stands for SQUID sensors :thinking:

Reviewer 3: adding comment to main thread to preserve visible history

> For these files, the `*_markers` suffix MUST be used.

Reviewer: We use `*_<suffix><ext>` to indicate a type of files, but I don't think we use `*_<suffix>` to indicate the suffix.

```suggestion
For these files, the `markers` suffix MUST be used.
```
